tensorboard/webapp/metrics/testing.ts

```diff
@@ -286,7 +287,7 @@ export function provideMockCardSeriesData(
 ) {
   const cardMetadata = {...createCardMetadata(plugin), ...metadataOverride};
   const runId = cardMetadata.runId;
-  let runToSeries = null;
+  let runToSeries: RunToSeries | null = null;
```

Comment:
This seems out of place. Although it also seems like a good change.

Reply:
This is kinda out of place but my IDE was complaining about it and I didn't feel it warranted a standalone PR